ts_auto_deps/BUILD.bazel

```diff
@@ -16,3 +18,16 @@ go_binary(
     embed = [":go_default_library"],
     visibility = ["//visibility:public"],
 )
+
+filegroup(
```

Comment:
don't include any .go files in the distro. see recent commit: we now have pre-built binary for ts_auto_deps

Reply:
👍